api/scpca_portal/models/sample.py

```diff
@@ -149,7 +137,9 @@ def get_metadata(self) -> Dict:
 
     def get_lock_name(self, download_config: Dict) -> str:
         """Returns a unique lock name based on multiplexed status and download config"""
-        lock_prefix = self.scpca_id if not self.is_multiplexed else "_".join(self.multiplexed_ids)
+        lock_prefix = (
+            self.scpca_id if not self.has_multiplexed_data else "_".join(self.multiplexed_ids)
```

Comment:
I noticed this also, but I was concerned that it might be confusing. It feels non-obvious that self.multiplexed_ids would return the normal self.scpca_id if its a non-multiplexed sample. It's obvious when you look up at the implementation, but wasn't sure if it would lead to confusion. Thoughts?

Reply:
I think this is ok to still use it for now, we should probably rework the name to make it more obvious that it is the scpca id as a list that will contain mutliplexed ids. This is for internal use only so we have some flexibility here to work with.